mula/.ci/docker-compose.yml

```
@@ -47,3 +50,44 @@ services:
       test: [ "CMD", "rabbitmqctl", "status" ]
       interval: 5s
       retries: 4
+
+  ci_bytes:
+    build:
+      context: ../bytes
+      dockerfile: Dockerfile
+      target: dev
+      args:
+        ENVIRONMENT: dev
+    command: uvicorn bytes.api:app --host 0.0.0.0 --port 8000 --reload --reload-dir /app/bytes/bytes
+    ports:
+      - "127.0.0.1:8002:8000"
+    env_file:
+      - .ci/.env.test
+    volumes:
+      - ../bytes/:/app/bytes
+      - bytes-data:/data
+    depends_on:
+      ci_postgres:
+        condition: service_healthy
+
+  ci_katalogus:
+    build:
+      context: ..
+      dockerfile: boefjes/Dockerfile
+      target: dev
+      args:
+        ENVIRONMENT: dev
+    command: uvicorn boefjes.katalogus.api:app --host 0.0.0.0 --port 8000 --reload --reload-dir /app/boefjes/boefjes/katalogus
```

- `8000` is already the default port, which makes the option unnecessary
- The `--reload` and `--reload-dir` options don't make sense within this (pipeline) context and potentially slow down this service while being tested

```suggestion
    command: uvicorn boefjes.katalogus.api:app --host 0.0.0.0
```